src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -6,14 +6,14 @@
 
     <services>
         <service id="translator.default" class="Symfony\Bundle\FrameworkBundle\Translation\Translator">
-            <argument type="service" id="service_container" />
+            <argument type="service-locator" /> <!-- translation loaders -->
             <argument type="service" id="translator.selector" />
-            <argument type="collection" /> <!-- translation loaders -->
+            <argument>%kernel.default_locale%</argument>
```

Comment:
The argument exists in the parent constructor with getter/setter, I find consistent to have it here. An option would be fine too